src/pages/workspace/accounting/qbo/advanced/QuickbooksAccountSelectPage.tsx

```diff
@@ -0,0 +1,100 @@
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import type {ListItem} from '@components/SelectionList/types';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import AdminPolicyAccessOrNotFoundWrapper from '@pages/workspace/AdminPolicyAccessOrNotFoundWrapper';
+import FeatureEnabledAccessOrNotFoundWrapper from '@pages/workspace/FeatureEnabledAccessOrNotFoundWrapper';
+import PaidPolicyAccessOrNotFoundWrapper from '@pages/workspace/PaidPolicyAccessOrNotFoundWrapper';
+import withPolicy from '@pages/workspace/withPolicy';
+import type {WithPolicyProps} from '@pages/workspace/withPolicy';
+import * as Policy from '@userActions/Policy';
+import CONST from '@src/CONST';
+
+type SelectorType = ListItem & {
+    value: string;
+};
+
+// TODO: remove once UI is approved
+const DRAFT = [
+    {name: 'Croissant Co Payroll Account', id: 'Croissant Co Payroll Account'},
+    {name: 'Croissant Co Money in Clearing', id: 'Croissant Co Money in Clearing'},
+    {name: 'Croissant Co Debts and Loans', id: 'Croissant Co Debts and Loans'},
+];
+
+function QuickbooksAccountSelectPage({policy}: WithPolicyProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const selectedAccount = DRAFT[0].id; // selected
+
+    const policyID = policy?.id ?? '';
+    const {bankAccounts, creditCards} = policy?.connections?.quickbooksOnline?.data ?? {};
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+    const accountOptions = useMemo(() => DRAFT || [...(bankAccounts ?? []), ...(creditCards ?? [])], [bankAccounts, creditCards]);
+
+    const qboOnlineSelectorOptions = useMemo<SelectorType[]>(
+        () =>
+            accountOptions?.map(({id, name}) => ({
+                value: id,
+                text: name,
+                keyForList: id,
+                isSelected: selectedAccount === id,
+            })),
+        [selectedAccount, accountOptions],
+    );
+    const listHeaderComponent = useMemo(
+        () => (
+            <View style={[styles.pb2, styles.ph5]}>
+                <Text style={[styles.pb5, styles.textNormal]}>{translate('workspace.qbo.advancedConfig.invoiceAccountSelectDescription')}</Text>
+            </View>
+        ),
+        [translate, styles.pb2, styles.ph5, styles.pb5, styles.textNormal],
+    );
+
+    const initiallyFocusedOptionKey = useMemo(() => qboOnlineSelectorOptions?.find((mode) => mode.isSelected)?.keyForList, [qboOnlineSelectorOptions]);
+
+    const saveSelection = useCallback(
+        ({value}: SelectorType) => {
+            Policy.updatePolicyConnectionConfig(policyID, CONST.QUICK_BOOKS_CONFIG.REIMBURSEMENT_ACCOUNT_ID, value);
+            Navigation.goBack();
```

Comment:
Can you please add a fallback route here so this works correctly on deep link

(do same with other pages)